lib/fluent/plugin/in_sample.rb

```diff
@@ -39,6 +39,8 @@ class SampleInput < Input
     config_param :auto_increment_key, :string, default: nil
     desc "The boolean to suspend-and-resume incremental value after restart"
     config_param :suspend, :bool, default: false,deprecated: 'This parameters is ignored'
+    desc "It force to re-generate a new record by default. You can disable it if filter does not do destructive change"
+    config_param :renew_record, :bool, default: true
```

Comment:
If possible, I'd like to invert the meaning and set the default value to `false`.
It is because Fluentd's `bool` value can be omitted and true/false can be represented just by the presence or absence of a config name.

Reply:
`disable_copy`, for example?

Reply:
Thanks, fixed.

Reply:
Oops, I've overlooked your comment.